### PR TITLE
Update comfig.cfg

### PR DESCRIPTION
**Pull request by Tiagoquix**

### CODE REVIEW

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 net_maxpacketdrop 0 // Use all packets
 //net_maxpacketdrop 5000 // Use default packet drop threshold
-cl_timeout 65 // Max client timeout
+cl_timeout 60 // Max client timeout
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/comfig.cfg:21]: How much better is 60 vs 65?

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 2]
```diff
@@ -48,8 +48,8 @@ alias interp_standard_lan "cl_interp .0151515"
 alias packet_rate_congestion_aliases "alias interp_unreliable interp_congestion_unreliable;alias interp_safe interp_congestion_safe;alias interp_balanced interp_congestion_balanced;alias interp_lan interp_congestion_lan"
 alias packet_rate_standard_aliases "alias interp_unreliable interp_standard_unreliable;alias interp_safe interp_standard_safe;alias interp_balanced interp_standard_balanced;alias interp_lan interp_standard_lan"
 
-alias packet_rate_congestion "cl_cmdrate 33;cl_updaterate 32.9999999;packet_rate_congestion_aliases"
-alias packet_rate_standard "cl_cmdrate 66;cl_updaterate 65.9999999;packet_rate_standard_aliases"
+alias packet_rate_congestion "cl_cmdrate 33;cl_updaterate 33;packet_rate_congestion_aliases"
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/comfig.cfg]: This is intentional. I'm currently experimenting with an off by one error caused by floating point usage in the networking system.

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 3]
```diff
@@ -598,17 +598,17 @@ alias props_ultra "r_decalstaticprops 1;cl_phys_props_enable 1;cl_phys_props_max
 // Bodies that spawn on death with adjustable physics simulation and fading
 
 //cl_ragdoll_fade_time 15 // Fade out ragdolls in 15 seconds
-//cl_ragdoll_fade_time 5 // Disables ragdoll effects
+//cl_ragdoll_fade_time 5 // Fade out ragdolls in 5 seconds
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/comfig.cfg]: Having it higher than 5 enables certain ragdoll effects.

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 4]
```diff
@@ -598,17 +598,17 @@ alias props_ultra "r_decalstaticprops 1;cl_phys_props_enable 1;cl_phys_props_max
...
 //cl_ragdoll_forcefade 1 // Effectively disables ragdolls by instantly fading them
 //cl_ragdoll_physics_enable 0 // Disable ragdoll physics, where most of the performance from ragdolls comes from
 //cl_ragdoll_physics_enable 1 // Enable ragdolls physics for meaningful ragdolls
 //ragdoll_sleepaftertime 0 // Instantly sleep ragdolls
 //ragdoll_sleepaftertime 1.5 // Wait a reasonable time before sleeping ragdoll physics
 
-alias ragdolls_off "cl_ragdoll_fade_time -1;cl_ragdoll_forcefade 1;cl_ragdoll_physics_enable 0;ragdoll_sleepaftertime 0"
+alias ragdolls_off "cl_ragdoll_fade_time 0;cl_ragdoll_forcefade 1;cl_ragdoll_physics_enable 0;ragdoll_sleepaftertime 0"
 alias ragdolls_medium "cl_ragdoll_fade_time 5;cl_ragdoll_forcefade 0;cl_ragdoll_physics_enable 1;ragdoll_sleepaftertime 1.5"
-alias ragdolls_high "cl_ragdoll_fade_time 5.1;cl_ragdoll_forcefade 0;cl_ragdoll_physics_enable 1;ragdoll_sleepaftertime 3.0"
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/comfig.cfg:611]: Having a lower time optimizes the ragdolls.

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 5]
```diff
@@ -63,7 +63,7 @@ alias snapshots_safe "cl_interp_ratio 2;interp_safe"
 alias snapshots_balanced "cl_interp_ratio 1;interp_balanced"
 alias snapshots_lan "cl_interp_ratio 1;interp_lan"
 
-alias snapshots snapshots_balanced
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/comfig.cfg]: This is intentional. I am testing a unified interp again.

**Review by mastercoms — CHANGES_REQUESTED** [review]

File: config/mastercomfig/cfg/comfig/comfig.cfg (hunk 1 above)
Comment (Tiagoquix) [reply config/mastercomfig/cfg/comfig/comfig.cfg:21]: Less time to disconnect.

File: config/mastercomfig/cfg/comfig/comfig.cfg (hunk 4 above)
Comment (Tiagoquix) [reply config/mastercomfig/cfg/comfig/comfig.cfg:611]: Alright, but remove the ".0" from ragdoll_sleepaftertime, it's unnecessary to keep it.

File: config/mastercomfig/cfg/comfig/comfig.cfg (hunk 1 above)
Comment (mastercoms) [reply config/mastercomfig/cfg/comfig/comfig.cfg:21]: My thinking is that people can manually disconnect if they want to.

**Review by mastercoms — APPROVED** [review]

**Comment by mastercoms** [comment]
Thank you for your contribution to mastercomfig!

**Pull request merged**